src/resources/character.py

```diff
@@ -1,24 +1,55 @@
-from rich.text import Text
+from threading import Thread
```

Comment:
What is the reason for using threads?

Reply:
The reason for threading is so that movement controls which run under a ``while True`` loop can be checking for keyboard movements without it being blocking to the overall code

Reply:
fair enough